### DISCUSSION TITLE
Ability to see your boosts in game?  Feature request

### DISCUSSION THREAD

**Discussion by Alaindrea** [sunflower-land/sunflower-land#2934] [Ideas]
I would like a feature where you could see what boots the items on your farm (and that you are currently wearing) give to you. 

Perhaps this is a new building such as a wardrobe with a mirror or maybe this is just automatically available in a tab when you click your character. A "farm status" tab, perhaps?

Specifically, I would like to see the boots you have listed and where they are coming from specific skill, Item, or wearable. They should be categorized by what is boosted and the kind of boost it is, i.e. quantity or speed. It would be even better if you could see your chances of things such as hatching a mutant chicken. It would make it much easier for non-coders to understand what their items do and what items they would like to obtain.

**Comment by brian-man** [comment]
A contributor started building something like this but abandoned it because:
> This is an awesome initiative mate! I do want to mention that we will be building this functionality/UI for our next season. We can take some inspiration from what you have here but I would hate for you to put in a ton of time without knowing that we will be building this

-- @spencerdezartsmith

<https://github.com/sunflower-land/sunflower-land/pull/2926#issuecomment-1738384740>

FYI
